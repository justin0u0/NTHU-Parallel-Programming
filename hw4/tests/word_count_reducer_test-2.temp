0 cat 1
0 cat 1
0 cat 1
0 dog 1
0 dog 1
0 dog 1
1 egg 1
1 gas 1
1 horse 1
1 horse 1
1 horse 1
1 horse 1
1 horse 1
1 king 1
1 king 1
1 king 1
1 world 1
1 world 1