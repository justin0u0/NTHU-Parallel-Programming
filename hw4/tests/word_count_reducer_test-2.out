egg 3
frog 6
gas 2
horse 13
king 3
world 4
